src/sentry/integrations/github/tasks/codecov_account_link.py

```diff
@@ -31,6 +34,32 @@
 def codecov_account_link(
     integration_id: int,
     organization_id: int,
+) -> None:
+    link_codecov_account(integration_id, organization_id)
+
+
+@instrumented_task(
+    name="sentry.integrations.github.tasks.backfill_codecov_account_link",
+    queue="integrations.control.throttled",
+    max_retries=3,
+    silo_mode=SiloMode.CONTROL,
```

Comment:
Hrm, if these tasks are control bound, then my plan of using the limited pool won't work, as the workers there are region bound. We'll need to setup another processing pool for limited + control silo. I can get that started though.